rules-tests/TypeDeclaration/Rector/Class_/ReturnTypeFromStrictTernaryRector/Fixture/skip_union_void.inc

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace Rector\Tests\TypeDeclaration\Rector\Class_\ReturnTypeFromStrictTernaryRector\Fixture;
+
+class SkipUnionVoid
```

Comment:
test covers the edge case mention in https://github.com/rectorphp/rector/issues/8089#issuecomment-1651415299